prometheus-exporters/vrops-exporter/alerts/vcenter.alerts

```diff
@@ -2,7 +2,8 @@ groups:
 - name: vcenter.alerts
   rules:
   - alert: VcsaSslCertificateExpiryWarning
-    expr: vrops_vcenter_stats{statkey="vcsa_certificate_remaining_days"} < 31 and vrops_vcenter_stats{statkey="vcsa_certificate_remaining_days"} > 15
+    expr: vrops_vcenter_vcsa_certificate_remaining_days < 31 and 
+    vrops_vcenter_vcsa_certificate_remaining_days > 15
```

Comment:
Checking prometheus-exporters/vrops-exporter/alerts/vcenter.alerts
  FAILED:
prometheus-exporters/vrops-exporter/alerts/vcenter.alerts: yaml: line 6: could not find expected ':'